### PR TITLE
Annotatemyids: fix for header

### PR DESCRIPTION
**Pull request by mblue9**
Fix for https://github.com/galaxyproject/tools-iuc/issues/2319

FOR CONTRIBUTOR:
* [ ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/annotatemyids/annotateMyIDs.xml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<tool id="annotatemyids" name="annotateMyIDs" version="3.7.0">
+<tool id="annotatemyids" name="annotateMyIDs" version="3.7.0+galaxy1">
```
Comment (nsoranzo) [inline tools/annotatemyids/annotateMyIDs.xml]: Bugfix, no need to increment the version?

Comment (mblue9) [reply tools/annotatemyids/annotateMyIDs.xml]: Oh yes, I forgot! Have reverted now.

**Comment by mblue9** [comment]
Btw the last commit (before the wrapper version revert) passed locally but failed here with Travis with conda errors like below but guess lets see if it happens again...
```
2019-02-28 23:19:06,154 DEBUG [galaxy.tools.deps.conda_util] Executing command: /home/travis/conda/bin/conda create -y --quiet --override-channels --channel iuc --channel conda-forge --channel bioconda --channel defaults --name mulled-v1-59aba3e2a88fbb08e5ca224a8e9b2f1fed0944f8101fd65a1cf23a72029c0bc9 bioconductor-org.hs.eg.db=3.7.0 bioconductor-org.mm.eg.db=3.7.0 bioconductor-org.dm.eg.db=3.7.0 bioconductor-org.dr.eg.db=3.7.0
ERROR conda.core.link:_execute(543): An error occurred while installing package 'bioconda::bioconductor-org.dm.eg.db-3.7.0-r351_0'.
LinkError: post-link script failed for package bioconda::bioconductor-org.dm.eg.db-3.7.0-r351_0
running your command again with `-v` will provide additional information
location of failed script: /home/travis/conda/envs/mulled-v1-59aba3e2a88fbb08e5ca224a8e9b2f1fed0944f8101fd65a1cf23a72029c0bc9/bin/.bioconductor-org.dm.eg.db-post-link.sh
==> script messages <==
<None>

Attempting to roll back.
```

**Comment by nsoranzo** [comment]
Still failing, xref. https://github.com/bioconda/bioconda-recipes/issues/13846

**Comment by mblue9** [comment]
Ok, thanks for the info.

**Comment by bgruening** [comment]
Restarted, I think this issue is unrelated. I hope that BioC was just down.

**Comment by nsoranzo** [comment]
@mblue9 This seems to be a conda issue, which version do you have locally?

**Comment by mblue9** [comment]
I have conda 4.6.1

**Comment by nsoranzo** [comment]
Conda wget and openssl packages were causing this issue, new packages for both have been released and Travis is now green!

File: .travis.yml [hunk 2]
```diff
@@ -81,7 +81,7 @@ install:
   - pip install planemo
   - planemo conda_init
   - export PATH="$PLANEMO_CONDA_PREFIX/bin:$PATH"
-  - conda install -y -c conda-forge conda=4.6.4
+  - conda install -y -c conda-forge conda=4.5.12
```
Comment (nsoranzo) [inline .travis.yml]: We can revert this now, I think.

**Comment by bgruening** [comment]
Thanks all!

**Comment by mblue9** [comment]
Great! Thanks for sorting that.

**Pull request merged**